Sources/TokamakCore/Shapes/Path/Path.swift

```diff
@@ -0,0 +1,192 @@
+// Copyright 2020-2021 Tokamak contributors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+//  Created by Carson Katri on 06/28/2020.
+//
+
+import Foundation
+
+/// The outline of a 2D shape.
+public struct Path: Equatable, LosslessStringConvertible {
```

Comment:
This is a smaller version of the original `Path.swift` that satisfies linter's file length constraints.